solutions/remotemonitoring/single-vm/setup.sh

```diff
@@ -12,32 +12,42 @@ PKEY="${CERTS}/tls.key"
 
 # ========================================================================
 
-export HOST_NAME="${1:-localhost}"
-export APP_RUNTIME="${3:-dotnet}"
-export PCS_IOTHUB_CONNSTRING="$8"
-export PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING="$9"
-export PCS_TELEMETRY_DOCUMENTDB_CONNSTRING="$9"
-export PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING="$9"
-export PCS_IOTHUBREACT_ACCESS_CONNSTRING="$8"
-export PCS_IOTHUBREACT_HUB_NAME="${10}"
-export PCS_IOTHUBREACT_HUB_ENDPOINT="${11}"
-export PCS_IOTHUBREACT_HUB_PARTITIONS="${12}"
-export PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT="${13}"
-export PCS_IOTHUBREACT_AZUREBLOB_KEY="${14}"
-export PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX="${15}"
-export PCS_CERTIFICATE="${16}"
-export PCS_CERTIFICATE_KEY="${17}"
-export PCS_AZUREMAPS_KEY="${18}"
-export PCS_AUTH_ISSUER="https://sts.windows.net/${5}/"
-export PCS_AUTH_AUDIENCE="$6"
+export HOST_NAME="localhost"
+export PCS_LOG_LEVEL="Info"
+export APP_RUNTIME="dotnet"
```

Comment:
What happens if it's java? Does this get overwritten?

Reply:
Yes it taken from --runtime